vm/src/stdlib/os.rs

```diff
@@ -1229,7 +1229,7 @@ fn extend_module_platform_specific(vm: &VirtualMachine, module: PyObjectRef) ->
         "openpty" => ctx.new_rustfunc(os_openpty),
     });
 
-    #[cfg(not(target_os = "macos"))]
+    #[cfg(not(any(target_os = "macos", target_os = "android", target_os = "redox")))]
```

Comment:
Is this a blacklist instead of a whitelist(maybe linux)?

Reply:
It's actually [very specific](https://docs.rs/nix/0.15.0/src/nix/unistd.rs.html#962-987).